site/zenodo_rdm/support/errors.py

```diff
@@ -0,0 +1,20 @@
+from ..errors import ZenodoRDMError
+
+
+class ConfirmationEmailNotSent(ZenodoRDMError):
+    def __init__(self, message):
+        """Initialise error."""
+        super().__init__(
+                "Confirmation email failed to send. Error: {}".format(message)
+        )
+
+class SupportEmailNotSent(ZenodoRDMError):
+    def __init__(self, message):
+        """Initialise error."""
+        super().__init__(
+                "Support email failed to send. Error: {}".format(message)
+        )
+
+
+class FormValidationError(ZenodoRDMError):
```

Comment:
Currently not in use. Can be used when Marshmallow fails to validate the form data  or simply removed (marshmallow already throws a `ValidationError`) .